licenses/BSL.txt

```diff
@@ -7,16 +7,22 @@ Licensor:             Rocicorp, LLC
 Licensed Work:        replicache 6.2.0
                       The Licensed Work is (c) 2020 Rocicorp, LLC.
 
-Additional Use Grant: You may use the Licensed Work when your application uses
-                      the Licensed Work with a total of less than 100 client
-                      instances for any purpose.
+Additional Use Grant: You may make use of the Licensed Work, provided you
+                      do not use it in any of the following ways:
+
+                      * Create a synchronization engine. A "synchronization
+                     engine" is a commercial or non-commercial offering that
+                     synchronizes data between two or more computers.
+
+                     * Use it for commercial purposes with more than 500
+                     monthly active client instances.
 
 Change Date:          2023-04-12
 
 Change License:       Apache License, Version 2.0
 
-For information about alternative licensing arrangements for the Software,
-please visit: https://replicache.dev/
+In order to use Replicache for commercial purposes with more than 500 monthly
+active client instances, see: https://replicache.dev/#price.
```

Comment:
I know there is a contact section just below https://replicache.dev/#price but i think we should have a sentence in the price section that tells them to email us to sign up for a commercial plan.

Reply:
I don't like having something so salesy in the license. Skipping this.